website/client/src/components/challenges/challengeDetail.vue

```diff
@@ -384,6 +384,17 @@ export default {
       memberResults: [],
     };
   },
+  watch: {
+    challenge: {
+      handler (newVal) {
+        this.$store.dispatch('common:setTitle', {
+          section: this.$t('challenge'),
+          subSection: newVal.name,
+        });
+      },
+      deep: true,
```

Comment:
instead of using `deep` you can limit the watcher to `challenge.name`:

```
watch: {
    'challenge.name': {
      handler (newVal) {
        this.$store.dispatch('common:setTitle', {
          section: this.$t('challenge'),
          subSection: newVal.name,
        });
      },
  }
}
```